### PR TITLE
chore: split up GitHub Actions

### PR DESCRIPTION
**Pull request by tujoworker**
In order to lower the CI duration, this PR splits up the actions into several workflows. Now only some actions do use macos when running the visual tests and during icon fetch.
We also cache all `node_modules` on ubuntu directly and safe with that some seconds. A quick test shows that the same does take over 8 min on macos, so not an option.

The visual tests do only run when the commit does contain files inside `./packages/dnb-eufemia/src/**/*`. So they will not run when only docs are updated. Is this good?

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+      - name: Release
+        if: (github.ref == 'refs/heads/release' ||
+          github.ref == 'refs/heads/beta' ||
+          github.ref == 'refs/heads/alpha')
+        run: yarn workspace @dnb/eufemia publish:ci
+
+      - name: Slack
+        uses: 8398a7/action-slack@v3
+        with:
+          channel: '#eufemia-dev'
```
Comment (dinarosv) [inline .github/workflows/release.yml:95]: Denne eufemia-dev slack kanalen er en lukket gruppe?

Comment (tujoworker) [reply .github/workflows/release.yml:95]: ja. Jeg la deg til 👍

**Review by dinarosv — APPROVED** [review]
Very good work 👏  

How much time did we save on this?

**Comment by tujoworker** [comment]
Depends. When the visual tests run, then we are stil on 15 min – else we are down on 5 min

**Pull request merged**